src/modules/Shows.test.js

```diff
@@ -0,0 +1,24 @@
+import fetchMock from "jest-fetch-mock";
+const Shows = require("./Shows"); 
+// require('jest-fetch-mock');
+
+
+describe('Shows', () => {
+    test('should return the correct count of items', async () => {
+      // Mock the fetch response
+      const mockResponse = [{ id: 1, title: 'Episode 1' }, { id: 2, title: 'Episode 2' }, { id: 3, title: 'Episode 3' }];
+      global.fetch = jest.fn().mockResolvedValue({
+        json: jest.fn().mockResolvedValue(mockResponse)
+      });
+  
```

Comment:
Nice Job on mocking the response, it would be better if you mock it as it appears in the original API.